packages/core/src/components/breadcrumb/breadcrumb.css

```diff
@@ -0,0 +1,71 @@
+.hds-breadcrumb {
+  display: flex;
+  padding: var(--spacing-s) 0;
+}
+
+.hds-breadcrumb__list,
+.hds-breadcrumb__list {
+  flex-direction: row;
+  align-items: center;
+  word-wrap: break-word;
+}
+
+.hds-breadcrumb__list--mobile {
+  display: flex;
+}
+
+.hds-breadcrumb__list--desktop {
+  display: none;
+}
+
+.hds-breadcrumb__list-item {
+  align-items: center;
+  display: flex;
+  line-height: var(--lineheight-l);
+}
+.hds-breadcrumb__list-item--active {
+  font-weight: 700;
+  /* to match 3px border around links */
+  padding: 3px;
+}
+
+.hds-breadcrumb__link.hds-breadcrumb__link {
+  --link-visited-color: none;
+  --link-color: var(--color-black-90);
+}
+
+.hds-breadcrumb__link:hover,
+.hds-breadcrumb__link:focus {
+  text-decoration-color: var(--color-black-90);
+}
+
+.hds-breadcrumb__list--mobile .hds-breadcrumb__link {
+  font-size: var(--fontsize-body-l);
+}
+
+.hds-breadcrumb__back-arrow.hds-breadcrumb__back-arrow {
+  display: inline-flex;
+  /* compensate the white space in the icon */
+  margin-left: -5px;
+  margin-right: -3px;
```

Comment:
Icons should not have white space to avoid this kind of "fine-tuning", but as long as it is there, I think this is mandatory  to align the icon correctly.